_docs/examples/provider/kustomization.yaml

```diff
@@ -1,15 +1,16 @@
 bases:
-  - github.com/ovrclk/akash/_docs/kustomize/akash-provider?ref=master
-  # - ../../kustomize/akash-provider
+  - ../../kustomize/akash-provider
+    #- github.com/ovrclk/akash/_docs/kustomize/akash-provider?ref=master
 
-images:
-  - name: ovrclk/akashctl
+namespace: akash-services
 
+images:
+  - name: ovrclk/akash
     ##
-    # akashctl version
+    # akash version
     ##
 
-    newTag: latest
+    newTag: 0.7.21-doubletap.rc21
```

Comment:
have we released `0.9.x` yet?

Reply:
We do! Now the question; are we safe to set it to `latest`?